var/spack/repos/builtin/packages/spectrum-mpi/package.py

```diff
@@ -0,0 +1,82 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at International Business Machines Corporation
+#
+# This file is part of Spack.
+# Created by Serban Maerean, serban@us.ibm.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class SpectrumMpi(Package):
+    """
+    IBM MPI implementation from Spectrum MPI.
+
+    """
+
+    homepage = "http://www.example.com"
+    url      = "http://www.example.com/ibm-mpi-10.1.0.2.tar.gz"
+
+    provides('mpi')
+
+    def install(self, spec, prefix):
+        raise InstallError('IBM MPI is not installable; it is vendor supplied')
+
+    def setup_dependent_package(self, module, dspec):
+        # get library name and directory
+        self.spec.mpi_base_dir = self.prefix
```

Comment:
current `mpi` packages only set (e.g. `openmpi`)
```
self.spec.mpicc = join_path(self.prefix.bin, 'mpicc')
        self.spec.mpicxx = join_path(self.prefix.bin, 'mpic++')
        self.spec.mpifc = join_path(self.prefix.bin, 'mpif90')
        self.spec.mpif77 = join_path(self.prefix.bin, 'mpif77')
        self.spec.mpicxx_shared_libs = [
            join_path(self.prefix.lib, 'libmpi_cxx.{0}'.format(dso_suffix)),
            join_path(self.prefix.lib, 'libmpi.{0}'.format(dso_suffix))
        ]
```
so i would not add anything else as it would introduce discrepancies between different MPIs.